src/components/Images.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import image1 from '../../assets/suchar1.jpg';
+import image2 from '../../assets/suchar2.jpg';
+import image3 from '../../assets/suchar3.jpg';
+
+import styles from './Images.css';
+import commonStyles from '../common.css';
+
+/* ????
+function importAll(r) {
+    let images = {};
+    r.keys().map((item, index) => { images[item.replace('./', '')] = r(item); });
+    return images;
+}
+
+const images = importAll(require.context('./images', false, '/\.png/'));
+
+<img src={images['0.png']} />
+ */
+const Images = () => {
+    return (
+        <div className={`${commonStyles.column} ${styles.imageContainer}`}>
```

Comment:
You should take a look at 'classnames' library, pretty commonly used:
https://github.com/JedWatson/classnames